### PR TITLE
 [fix](fe-core) TestWithFeService do not clean up dorisHome

### PR DESCRIPTION
**Pull request by dinggege1024**
# Proposed changes

Issue Number: #13069 

## Problem summary

alter clean dir

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [✅] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [✅] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [✅] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [✅] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [✅] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by stalary** [comment]
Please check fe ut.

**Comment by dinggege1024** [comment]
> Please check fe ut.

Why my local build test fine, but ci could not.

I used this command:
```bash
mvn clean install -Dos.arch=x86_64 -Dcheckstyle.skip -pl fe-core -am
```

**Comment by dinggege1024** [comment]
@jackwener  Hi jack can you help this pr?

File: fe/fe-core/src/test/java/org/apache/doris/utframe/TestWithFeService.java [hunk 1]
```diff
@@ -540,4 +547,19 @@ private void checkAlterJob() throws InterruptedException, MetaNotFoundException
         }
     }
 
+    // clear the specified dir
```
Comment (jackwener) [inline fe/fe-core/src/test/java/org/apache/doris/utframe/TestWithFeService.java:550]: Why not use FileUtils.deleteDirectory

Comment (dinggege1024) [reply fe/fe-core/src/test/java/org/apache/doris/utframe/TestWithFeService.java:550]: > Why not use FileUtils.deleteDirectory

If so, Exception, [unable-to-delete-directory](https://stackoverflow.com/questions/50532263/java-delete-directory-ioexception-unable-to-delete-directory) will be thrown.

So i reference [createAndClearDir](https://github.com/apache/doris/blob/0dd2fb758cc777284b4506d8ceaa51fc4603a5cf/fe/fe-core/src/test/java/org/apache/doris/utframe/MockedFrontend.java#L151)

**Review by jackwener — APPROVED** [review]
LGTM

**Pull request merged**